app/controllers/integrations/base_controller.rb

```diff
@@ -51,6 +51,7 @@ def create
 
     json = {
       deploy_ids: deploys.map(&:id).compact,
+      deploy_options: options,
```

Comment:
I was kinda expecting `release: release` ?
... so return something we have a serializer for instead of some internal options hash
... but either way is fine, the part you are interested in is unlikely to change much

Reply:
Yeah, i wasn't sure what to do.  Since the `release` isn't always there.   But to be honest I would rather do it the way you suggest, so i'll update the PR